kbase-extension/static/kbase/js/widgets/function_output/modeling/KBModeling.js

```diff
@@ -44,7 +44,7 @@ function KBModeling(token) {
             method = 'fbaModelServices.'+method;
             return call_ajax(url, method, params)
         } else if (service == 'biochem') {
-            s_url = 'https://kbase.us/services/service_wizard';
+            s_url = window.kbconfig.urls.service_wizard || 'https://ci.kbase.us/services/service_wizard';
```

Comment:
Just making a(nother?) note that we need to eventually update these widgets to get loaded with require like everything else. Then we can just include the config module and get URLs from there, instead of depending on a global variable that may or may not exist.

But for now, this is fine.

Reply:
I don't know much about require.js but if we get a refactor of these viewers in the roadmap I'd take a look at it.

Reply:
Sure. It'll be some work to do, mainly since all of the modeling viewer widgets depend pretty heavily on each other being in the global namespace. It's not super high priority yet (though it will be), so don't worry about it for now.